python/unitytrainers/ppo/models.py

```diff
@@ -44,6 +49,92 @@ def create_reward_encoder():
         update_reward = tf.assign(last_reward, new_reward)
         return last_reward, new_reward, update_reward
 
+    def create_curiosity_encoders(self):
```

Comment:
Maybe for latter : Each encoder could be in its own script ?

Reply:
Can you elaborate? Are you thinking of a `curiosity.py`?